MH4U Dex/MonsterDetailsViewController.m

```diff
@@ -29,39 +34,61 @@ @implementation MonsterDetailsViewController
 - (void)viewDidLoad
 {
     [super viewDidLoad];
-    [self.lowRankSubView setHidden:YES];
+    self.overviewSubView.hidden = NO;
+    self.lowRankSubView.hidden = YES;
+    self.highRankSubView.hidden = YES;
+    self.GRankSubView.hidden = YES;
 }
 
 - (IBAction)segmentedControlChanged:(UISegmentedControl *)segmentedControl
 {
     switch (segmentedControl.selectedSegmentIndex) {
         case Overview:
-            [self.overviewSubView setHidden:NO];
-            [self.lowRankSubView setHidden:YES];
+            self.overviewSubView.hidden = NO;
+            self.lowRankSubView.hidden = YES;
+            self.highRankSubView.hidden = YES;
+            self.GRankSubView.hidden = YES;
             self.navigationItem.title = [NSString stringWithFormat:@"%@ Overview", self.monster];
             break;
         case LowRank:
-            [self.overviewSubView setHidden:YES];
-            [self.lowRankSubView setHidden:NO];
+            self.overviewSubView.hidden = YES;
+            self.lowRankSubView.hidden = NO;
+            self.highRankSubView.hidden = YES;
+            self.GRankSubView.hidden = YES;
             self.navigationItem.title = [NSString stringWithFormat:@"%@ LR Drops", self.monster];
             break;
+        case HighRank:
+            self.overviewSubView.hidden = YES;
+            self.lowRankSubView.hidden = YES;
+            self.highRankSubView.hidden = NO;
+            self.GRankSubView.hidden = YES;
+        case GRank:
+            self.overviewSubView.hidden = YES;
+            self.lowRankSubView.hidden = YES;
+            self.highRankSubView.hidden = YES;
+            self.GRankSubView.hidden = NO;
         default:
             break;
     }
```

Comment:
It might be more concise to set all four `.hidden` properties to `YES` above the `switch`-`case` statement, so that you only have to set one `.hidden` property to `NO` in each case.